### DISCUSSION TITLE
What does the TUN address and gateway 10.8.1.1 mean?

### DISCUSSION THREAD

**Discussion by dickdevries** [RaspAP/raspap-webgui#1134] [Q&A]
Hi,

I am a little bit confused ( I'm not a pro). Did the upgrade from v2.7.4 to 2.8.3. Upgrade process was smooth.
I use OpenVPN with NordVPN, see the IP address I'm connected to and did an IP check (Whatismyip.com) result is Correct.
What confuses me is when I check 'Network' I see the following:
![afbeelding](https://user-images.githubusercontent.com/33785543/157067890-8f56710e-289f-42ef-8be7-f9daa0a101e1.png)
The static IP address of RaspAP is 10.3.141.1 and my clients get by DHCP 10.3.141.50 - 255.
What does the TUN address 10.8.1.5 and gateway 10.8.1.1 mean? 
This was different in the previous version (2.7.4) I was using.

**Comment by billz** [comment]
`tun0` is a virtual network interface created for routing purposes. In this case, it listens for client connections on whichever port you (or your OpenVPN provider) has configured, and distributes virtual addresses to connecting clients from the 10.8.1.0/24 subnet.
